modules/network-observability-loki-install.adoc

```diff
@@ -5,7 +5,7 @@
 :_content-type: PROCEDURE
 [id="network-observability-loki-installation_{context}"]
 = Installing the Loki Operator
-It is recommended to install Loki using the link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.6], This version provides the ability to create a LokiStack instance using the `openshift-network` tennant configuration mode. It also provides fully automatic, in-cluster authentication and authorization support for Network Observability.
+It is recommended to install Loki using the link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.7], This version provides the ability to create a LokiStack instance using the `openshift-network` tenant configuration mode. It also provides fully automatic, in-cluster authentication and authorization support for Network Observability.
```

Comment:
> It is recommended to install Loki using the Loki Operator version 5.7

I know this is existing text. But, this confuses me. This seems to suggest that I install the Loki Operator, then use the Operator to install Loki. But a following sentence suggests that installing the Operator install Loki  (I think). 

> There are several ways you can install Loki. One way you can install the Loki Operator is by using...

This paragraph goes between installing Loki and the Loki Operator. Can I bother you to clean this up, assuming I'm not mixed up here?

Reply:
Addressed, I think I don't need that "using the" I can just say "It is recommended to install the Loki Operator version 5.7. Good catch.